src/constants/keyboard.lisp

```diff
@@ -2,6 +2,7 @@
 
 ;; Keycodes
 (defcenum keycodes
+    (:|none| 0)
```

Comment:
LGTM, quick note that indentation conventionally should be 2 spaces in this situations. I'll tack on a quick change for that right after this.

(Likewise L142 had whitespace modified)